packages/block-editor/src/components/alignment-control/stories/aliginment-toolbar.story.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { AlignmentToolbar } from '..';
+
+/**
+ * The `AlignmentToolbar` component renders a dropdown menu that displays alignment options for the selected block in `Toolbar`.
+ */
+const meta = {
+	title: 'BlockEditor/AlignmentToolbar',
+	component: AlignmentToolbar,
+	argTypes: {
+		value: {
+			control: { type: null },
+			defaultValue: 'undefined',
+			description: 'The current value of the alignment setting.',
+		},
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+			description:
+				"A callback function invoked when the toolbar's alignment value is changed via an interaction with any of the toolbar's buttons. Called with the new alignment value (ie: `left`, `center`, `right`, `undefined`) as the only argument.",
+		},
+	},
+};
+export default meta;
+
+const Template = ( { onChange, ...args } ) => {
+	const [ value, setValue ] = useState();
+	return (
+		<AlignmentToolbar
+			{ ...args }
+			onChange={ ( ...changeArgs ) => {
+				onChange( ...changeArgs );
+				setValue( ...changeArgs );
+			} }
+			value={ value }
+		/>
+	);
+};
+
+export const Default = Template.bind();
```

Comment:
This is fine, but just as a tip for easier authoring and maintenance, it will be a lot simpler to showcase the component in uncontrolled mode if the component supports it. It will literally be a one-liner 😄

```suggestion
export const Default = {};
```

Reply:
Wow, so easy! And more ideal. Thanks for letting me know!

Reply:
But then we can't reproduce the `onChange` moving on Storybook, but that's not a problem, right? Or I guess I might miss something...

Reply:
The `onChange` should still be observable in the Actions panel as before. Is it not working in your environment? The only reason we need to merge the incoming `onChange` function is when it's used in controlled mode and you need to call the `setValue` as well.

Reply:
In my environment, the story of the `AlignmentToolbar` component does not work properly. The `onChange` event is fired, but the text alignment is not updated. Does this component not support uncontrolled mode?

https://github.com/user-attachments/assets/989cc505-3b26-4789-a0ef-77540b31f42c

Reply:
Ah right, my bad! I guess it doesn't support uncontrolled. I'd still recommend using CSF 3 format (instead of the `Template.bind()`) for new stories though:

```js
export const Default = {
	render: function Template( { onChange, ...args } ) {
		const [ value, setValue ] = useState();
		return (
			<AlignmentToolbar
				{ ...args }
				onChange={ ( ...changeArgs ) => {
					onChange( ...changeArgs );
					setValue( ...changeArgs );
				} }
				value={ value }
			/>
		);
	},
};
```

Reply:
Thank you two! I updated as you wrote.